src/codegen/function-entry.lisp

```diff
@@ -82,11 +82,13 @@
                                     (alexandria:format-symbol *package* "A~D" i))
                               funs)))
       `(progn
-         #+sbcl
-         (declaim (sb-ext:start-block ,@funs))
-         ,@(reverse body)
-         #+sbcl
-         (declaim (sb-ext:end-block))))))
+         (declaim (inline ,@funs))
+         (locally (declaim (notinline ,@funs))
```

Comment:
does this work? I think I've only seen
```
(declaim inline)
;; code
(declaim notinline)
```

do we really want these funcs to always be inlined after their definition?

Reply:
It does not appear to actually work. I do think these functions, or at least the low numbered ones should be inlined. They are called on every indirect function call.